resources/views/forms/profile.blade.php

```diff
@@ -6,371 +12,389 @@
                 well as in other media associated with Eurofurence. You will be able to edit this information even after
                 the registration period has ended.</p>
         </div>
-        <div class="row mb-3">
-            <label class="col-sm-2 col-form-label fw-bold">Attend&shy;ance</label>
-        </div>
-        <div class="row mb-1">
-            <div class="col-sm-10 offset-sm-2">
-                <div class="form-check">
-                    <input class="form-check-input @error('attends_thu') is-invalid @enderror" name="attends_thu"
-                        @checked(old('_token') ? old('attends_thu') : $profile?->attends_thu === true) type="checkbox" id="attends_thu">
-                    <label class="form-check-label" for="attends_thu">
-                        {{config('con.day_2_name')}}
-                    </label>
+        @if ($applicationType === \App\Enums\ApplicationType::Share)
+            <input class="form-check-input" name="profile_hidden" role="switch" @checked(old('_token') ? old('profile_hidden') : $profile?->is_hidden === true) type="checkbox" id="dd-profile-hidden">
+            <label class="form-check-label" for="dd-profile-hidden">
+                <strong>Hide Profile</strong> – I wish to share the dealership's profile instead of providing my own.
```

Comment:
I think we generally use upper case for Dealership (though it's a bit inconsistent atm).

Reply:
As mentioned on the other PR, let's maybe do a separate issue and PR to fix the casing of that term in a consistent manner throughout the entire application. I'll create an issue for that. (=